From e63bc9f2356ccb9500368cdba99192b98732dbd4 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Thu, 12 Oct 2017 14:08:33 -0700
Subject: [PATCH] wmluiltok: add option to generate main()

wmluiltok fails to build due to lack of a main() function. Add an option
to the lex input to add it.
http://motif.ics.com/forum/compilation-error-no-main-wmluiltokc
---
 tools/wml/wmluiltok.l | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/wml/wmluiltok.l b/tools/wml/wmluiltok.l
index f02d9fe..b468095 100644
--- a/tools/wml/wmluiltok.l
+++ b/tools/wml/wmluiltok.l
@@ -1,3 +1,4 @@
+%option main
 %{
 /*
  * Motif
-- 
2.14.1

